core/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -469,6 +478,24 @@ public final GetResult get(Get get) throws EngineException {
     public abstract GetResult get(Get get, Function<String, Searcher> searcherFactory) throws EngineException;
 
     /**
+     * Return a {@code SeqID} representing the sequence ID for the given {@code Get}
+     * operation.
+     */
+    public final SeqID getSequenceID(Get get) throws EngineException {
```

Comment:
That's part of the reason I used a `Tuple` in the first place, I expect this method to go away in the future, though maybe not necessarily in this PR (see my comment on the PR)